src/Aspire.Hosting/Dashboard/TransportOptionsValidator.cs

```diff
@@ -0,0 +1,37 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Options;
+
+namespace Aspire.Hosting.Dashboard;
+
+internal class TransportOptionsValidator(IConfiguration configuration, DistributedApplicationExecutionContext executionContext) : IValidateOptions<TransportOptions>
+{
+    public ValidateOptionsResult Validate(string? name, TransportOptions options)
+    {
+        if (executionContext.IsPublishMode)
+        {
+            return ValidateOptionsResult.Success;
+        }
```

Comment:
I think we also want to noop if the dashboard isn't running.

Reply:
Even if we aren't running the dashboard, the resource server is still running.

Reply:
It's not 😄